behave.ini.dist-allscripts

```diff
@@ -10,5 +10,6 @@ patient=8501fd7d-4f46-4eca-a5c0-a5fd0112fe85
 strategy=refresh_token
 client_id=4e3ecd32-b17c-47ed-bbb0-ff60e870be1a
 client_secret=QYuz2cPx6e9Th/UkYrTwuBjChAYa8aebkT9VVsgC211NboKAqLt0BZwrxSp+De/oS4iSzUi2nePZI6NAF+UoIw==
-refresh_token=eyJhbGciOiJIUzI1NiIsImtpZCI6InRlbXAtZGVmYXVsdCIsInR5cCI6IkpXVCJ9.eyJuYmYiOjE0NjMwNjc4MDgsImV4cCI6MTQ5NDYwMzgwOCwiaWF0IjoxNDYzMDY3ODA4LCJ1bmlxdWVfbmFtZSI6IjRlM2VjZDMyLWIxN2MtNDdlZC1iYmIwLWZmNjBlODcwYmUxYSIsIlVzZSI6IlJlZnJlc2giLCJVc2VyIjoiODUwMWZkN2QtNGY0Ni00ZWNhLWE1YzAtYTVmZDAxMTJmZTg1Iiwic2NvcGUiOiJsYXVuY2gvcGF0aWVudCBwYXRpZW50LyoucmVhZCIsImlzcyI6Imh0dHBzOi8vbXVsb2dpbi5mb2xsb3dteWhlYWx0aC5jb20iLCJhdWQiOiJodHRwczovL211bG9naW4uZm9sbG93bXloZWFsdGguY29tIn0.OF-a--gMzZgqDugDE_TssFPIPxpmmI1Z6PohWfFBpHI
-url=https://fmhmulogin.azurewebsites.net/api/
+scope=launch/patient patient/*.read offline_access
+redirect_uri=https://tests.demo.syncfor.science/authorized/
```

Comment:
At this point, given we're passing these via POST and parsing fields out of them ourselves, these files might as well be JSON. (Am I missing something?)